### ISSUE TITLE
Element.append() does not work with defusedxml?

### ISSUE THREAD

**Issue by papadeltasierra** [tiran/defusedxml#43]
See the script below.  This works as expected.

    # from defusedxml.ElementTree import fromstring
    import xml.etree.ElementTree as ET
    
    THE_XML = "<ListOfItems></ListOfItems>"
    # xml = fromstring(THE_XML)
    xml = ET.fromstring(THE_XML)
    item = ET.Element('Item')
    xml.append(item)

However it I swap for the version that uses defusedxml then I get this error:

    Traceback (most recent call last):
    File "c:\temp\defusing.py", line 7, in <module>
        xml.append(item)
    TypeError: append() argument must be xml.etree.ElementTree.Element, not Element

Is this caused by defusedxml or is this some subtle difference between the C and Python implementations of xml.etree.Elementree?

**Comment by Aragahara** [comment]
I've had the same problem using this code
```python
import xml.etree.ElementTree
from defusedxml.ElementTree import parse

tree = parse('test.xml')
tree.getroot().append(xml.etree.ElementTree.Element("Test"))
```
with following error
```
Traceback (most recent call last):
  File "xmltest.py", line 5, in <module>
    tree.getroot().append(xml.etree.ElementTree.Element("Test"))
TypeError: append() argument must be xml.etree.ElementTree.Element, not Element
```
Got it working by importing only Element
```python
from xml.etree.ElementTree import Element
from defusedxml.ElementTree import parse

tree = parse('test.xml')
tree.getroot().append(Element("Test"))
```

**Comment by tiran** [comment]
The problem was fixed in #60. Please update to 0.7.1 or 0.8.0rc1.
